airbyte-webapp/src/utils/connectors/destinations.json

```diff
@@ -0,0 +1,50 @@
+{
```

Comment:
:information_source: This (and the sources.json) are JSON files, so I can import them in JS in the eslint rule, since I can't import TypeScript there, unless setting up a full tyepscript build for that plugin as well.

Reply:
Is there any chance these GUIDs change elsewhere?

Reply:
+1 to what Tim said - I don't believe these ever change and if they did it would break other parts of the system as well.